packages/lib/constants.ts

```diff
@@ -18,3 +18,4 @@ export const TRIAL_LIMIT_DAYS = 14;
 export const HOSTED_CAL_FEATURES = process.env.HOSTED_CAL_FEATURES || BASE_URL === "https://app.cal.com";
 /** @deprecated use `WEBAPP_URL` */
 export const NEXT_PUBLIC_BASE_URL = process.env.NEXT_PUBLIC_WEBAPP_URL || `https://${process.env.VERCEL_URL}`;
+export const PRISMA_CLIENT_CACHING_TIME = 1000 * 60 * 60 * 24;
```

Comment:
Let's keep this in the API

Reply:
Done, will move to @calcom/api/lib/constants.ts